documentation/modules/cruise-control/proc-tracking-cluster-rebalance.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-tracking-cluster-rebalance-{context}']
+
+= Tracking rebalances
+
+You can track the progress of a partition rebalance using status information in the `KafkaRebalance` resource.
+Tracking an active partition rebalance supports planning of cluster operations, including worker node maintenance, scaling, and broker upgrades. 
+Understanding key details, such as duration and the remaining data to transfer, helps with scheduling maintenance windows and assessing the impact of proceeding or canceling the rebalance.
+
+When a rebalance is in progress, Strimzi stores progress information in a `ConfigMap`.
+The `ConfigMap` is referenced by the `status.progress.rebalanceProgressConfigMap` property of the `KafkaRebalance` resource .
```

Comment:
... but IIRC it has a well defined name by a specific pattern (including the `KafkaRebalance` custom resource name). Maybe we can specify it here?

Reply:
The implementation ended up naming the `ConfigMap` with the same name as the `KafkaRebalance` resource it is referenced by. Added a note here in latest commit.